leakcanary-android-instrumentation/src/main/java/leakcanary/FailAnnotatedTestOnLeakRunListener.kt

```diff
@@ -0,0 +1,13 @@
+package leakcanary
+
+import org.junit.runner.Description
+
+class FailAnnotatedTestOnLeakRunListener: FailTestOnLeakRunListener() {
```

Comment:
Can you add javadoc and mention `@FailTestOnLeak` ?

Reply:
Done!